### PR TITLE
gRPC container coverage improvement

### PR DESCRIPTION
**Pull request by dmdashenkov**
- `Builder` methods test
 - `#addShutdownHook` test

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol PTAL

File: server/src/test/java/org/spine3/server/transport/GrpcContainerShould.java [hunk 1]
```diff
@@ -66,6 +77,59 @@ public void start_server() throws IOException {
         verify(server).start();
     }
 
+    @SuppressWarnings("MagicNumber")
+    @Test
+    public void add_and_remove_parameters_form_builder() {
```
Comment (armiol) [inline server/src/test/java/org/spine3/server/transport/GrpcContainerShould.java]: `from`.

File: server/src/test/java/org/spine3/server/transport/GrpcContainerShould.java [hunk 2]
```diff
@@ -66,6 +77,59 @@ public void start_server() throws IOException {
...
+
+        final Set<ServerServiceDefinition> serviceSet = builder.getServices();
+        assertSize(count, serviceSet);
+
+        final GrpcContainer container = builder.build();
+        assertNotNull(container);
+    }
+
+    @Test
+    public void add_shutdown_hook_to_runtime() throws NoSuchFieldException, IllegalAccessException, IOException {
```
Comment (armiol) [inline server/src/test/java/org/spine3/server/transport/GrpcContainerShould.java]: `public void stop_properly_upon_application_shutdown`.

In tests you tell what should happen, not how it is implemented.

File: server/src/test/java/org/spine3/server/transport/GrpcContainerShould.java [hunk 3]
```diff
@@ -66,6 +77,59 @@ public void start_server() throws IOException {
...
+        final Class<Runtime> runtimeClass = Runtime.class;
+        // private static Runtime currentRuntime
```
Comment (armiol) [inline server/src/test/java/org/spine3/server/transport/GrpcContainerShould.java]: Why commented code in the codebase?

File: server/src/main/java/org/spine3/server/transport/GrpcContainer.java [hunk 4]
```diff
@@ -180,6 +163,27 @@ public void run() {
         return builder.build();
     }
 
+    @VisibleForTesting
+    /* package */ Runnable getShutdownOperation() {
```
Comment (armiol) [inline server/src/main/java/org/spine3/server/transport/GrpcContainer.java]: `getOnShutdownCallback()`.

File: server/src/test/java/org/spine3/server/transport/GrpcContainerShould.java (hunk 3 above)
Comment (dmdashenkov) [inline server/src/test/java/org/spine3/server/transport/GrpcContainerShould.java]: This is to show the field signature. Taken from `java.lang.Runtime`.

**Comment by dmdashenkov** [comment]
@armiol PTAL

**Comment by armiol** [comment]
@dmdashenkov please resolve a conflict and feel free to merge.

**Pull request merged**